### ISSUE TITLE
Add a new class onto a trained pickle.

### ISSUE THREAD

**Issue by TheNarrator** [davidsandberg/facenet#485]
I'm successfully able to create a custom classifier with my own images. With **model.fit(emb_array, labels)** I am able to produce a classifier.

But whenever I add a person, I end up training a new classifier. Is it possible to add just **1** new person onto a loaded pickle without re-training the entire classifier? I think more or less asking if it's possible to take a softmax classifier with N persons, and make it N+1 without re-training the entire classifier.

**Comment by davidsandberg** [comment]
Maybe you can use a threshold on the distance between the embeddings. That way you would not have to retrain a classifier.

**Comment by blankxyz** [comment]
how to threshold on the distance between the embeddings. do u have any samples?

**Comment by elnasdgreat** [comment]
examples would be greatly appreciated!
